shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -1401,8 +1550,7 @@ abstract class JSTestBackend extends JSBackend(allowUnresolvedSymbols = false) {
     JSTestBackend.TestCode(SourceCode.fromStmts(polyfill.emit() ::: prelude).toLines, queries)
   }
 
-  private def generateNewDef(pgrm: Pgrm)(implicit scope: Scope, allowEscape: Bool): JSTestBackend.TestCode = {
-    
+  private def generateNewDef(pgrm: Pgrm, prettyPrintQQ: Bool)(implicit scope: Scope, allowEscape: Bool): JSTestBackend.TestCode = {
```

Comment:
Please don't needlessly change the whitespace, making the formatting inconsistent.
```suggestion
  private def generateNewDef(pgrm: Pgrm, prettyPrintQQ: Bool)(implicit scope: Scope, allowEscape: Bool): JSTestBackend.TestCode = {
  
```